client/blocks/comment-detail/index.jsx

```diff
@@ -109,6 +109,16 @@ export class CommentDetail extends Component {
 		setCommentStatus( commentId, postId, 'trash' === commentStatus ? 'approved' : 'trash' );
 	}
 
+	keyHandler = event => {
```

Comment:
Probably also that we have focus on the card, vs child elements too, otherwise we can't toggle back.